Parser/pegen/pegen.c

```diff
@@ -25,7 +25,7 @@ _PyPegen_add_type_comment(Parser *p, arg_ty a, char *tc)
     if (tc == NULL) {
         return a;
     }
-    PyObject *tco = _PyPegen_new_type_comment(p, tc);
+    PyObject *tco = NEW_TYPE_COMMENT(tc);
```

Comment:
Doesn't `NEW_TYPE_COMMENT` expect `tc` to be a `Token *`, i.e. that `tc->bytes` is a bytes object? I feel that `NEW_TYPE_COMMENT` would better be off as an inline function.